seqr/utils/elasticsearch/constants.py

```diff
@@ -294,23 +294,36 @@
     'StrVCTVRE_score': {'response_key': 'strvctvre'},
 }
 
-QUALITY_FIELDS = {'gq': 5, 'ab': 5, 'qs': 10}
-QUALITY_QUERY_FIELDS = deepcopy(QUALITY_FIELDS)
-QUALITY_QUERY_FIELDS.update({'gq_sv': 10})
+QUALITY_QUERY_FIELDS = {'gq_sv': 10}
+SHARED_QUALITY_FIELDS = {'gq': 5}
+SNP_QUALITY_FIELDS = {'ab': 5}
+SNP_QUALITY_FIELDS.update(SHARED_QUALITY_FIELDS)
+SV_QUALITY_FIELDS = {'qs': 10}
+SV_QUALITY_FIELDS.update(SHARED_QUALITY_FIELDS)
+QUALITY_QUERY_FIELDS.update(SNP_QUALITY_FIELDS)
+QUALITY_QUERY_FIELDS.update(SV_QUALITY_FIELDS)
+BASE_GENOTYPE_FIELDS_CONFIG = {
+    'sample_id': {},
+    'sample_type': {},
+    'num_alt': {'format_value': int, 'default_value': -1},
+}
 GENOTYPE_FIELDS_CONFIG = {
     'ad': {},
     'dp': {},
     'pl': {},
+}
+GENOTYPE_FIELDS_CONFIG.update(BASE_GENOTYPE_FIELDS_CONFIG)
+GENOTYPE_FIELDS_CONFIG.update({field: {} for field in SNP_QUALITY_FIELDS.keys()})
+SV_GENOTYPE_FIELDS_CONFIG = {
```

Comment:
I like the separated `SV_GENOTYPE_FIELDS_CONFIG`. But it causes quite a lot of changes to the tests.